src/page-response-impact/responseChartData.tsx

```diff
@@ -0,0 +1,57 @@
+import { zip } from "d3-array";
+import ndarray from "ndarray";
+
+import {
+  calculateCurves,
+  CurveData,
+  CurveFunctionInputs,
+} from "../infection-model";
+import { getAllValues, getColView } from "../infection-model/matrixUtils";
+import { seirIndex } from "../infection-model/seir";
+
+function calculateCurveData(facilitiesInputs: CurveFunctionInputs[]) {
+  return facilitiesInputs.map((facilityInput) => {
+    return calculateCurves(facilityInput);
+  });
+}
+
+function combineFacilitiesProjectionData(
+  facilitiesProjectionData: CurveData[],
+) {
+  if (!facilitiesProjectionData.length) return ndarray([], []);
+  const incarceratedData = facilitiesProjectionData.map(
+    (output) => output.incarcerated.data,
+  );
+  const staffData = facilitiesProjectionData.map((output) => output.staff.data);
+  const combinedData = zip(...incarceratedData, ...staffData);
+  const summedData = combinedData.map((row) => {
+    return row.reduce((sum, value) => {
+      return (sum += value);
+    }, 0);
+  });
+  return ndarray(summedData, [90, 9]);
```

Comment:
Nit: can we extract 90 and 9 out to well-named constants, so that future readers who are less well-versed with the domain model understand what they mean?

Reply:
Yes sounds good. I know 90 is the number of days, and 9 seems to be the number of SEIR categories, do these variables make sense? 

`const NUM_DAYS = 90; const NUM_SEIR_CATEGORIES = 9`

Reply:
this info helps me 😃